src/stores/index.ts

```diff
@@ -115,7 +115,7 @@ export const defaultHospitalSensor = derived(sensorList, (sensorList) => {
   return sensorList.find((d) => d.signal === 'confirmed_admissions_covid_1d_prop_7dav');
 });
 export const defaultDeathSensor = derived(sensorList, (sensorList) => {
-  return sensorList.find((d) => d.signal === 'deaths_7dav_incidence_prop');
+  return sensorList.find((d) => d.signal === 'deaths_covid_incidence_prop');
```

Comment:
Looks like we can remove this (and one or two above it); see comment in `src/modes/summary/Summary.svelte`